serving/README.md

```diff
@@ -1,4 +1,53 @@
-# Serving
+# Sepsis Detection As a Serverless API
+
+Instructions to deploy the sepsis-detection as a serverless function on OpenShit
+
+## To just deploy to Openshift
+
+To deploy and test on OpenShift
+
+1. Install OCP Serverless Operator from OperatorHub
+1. Install Knative Serving from Installed Operators under project: knative-serving
+
+```sh
+git clone https://github.com/redhat-naps-da/sepsis-detection.git
+cd serving
+oc project sepsis-detection
+oc apply -f fn/kubernetes/
+```
+
+This will create a knative service using. The minimum number of pods is set to 1 (not zero) due to 
```

Comment:
(completing the sentence) due to performance loading the model and pipeline each time from zero versus having at least 1 pod with the model and pipeline loaded

Reply:
fixed